File: sol/3477/3477.py
class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        n = len(fruits)
        unplaced = 0
        for f in fruits:
            for i, cap in enumerate(baskets):
                if cap >= f:
                    baskets.pop(i)
                    break
            else:
                unplaced += 1
        return unplaced

File: sol/3477/3477.cpp
struct Solution {
    int numOfUnplacedFruits(vector<int>& fruits, vector<int>& baskets) {
        int n = fruits.size(), unplaced = 0;
        for (int f : fruits){
            ++unplaced;
            for (int i = 0; i < n; ++i){
                if (baskets[i] >= f){
                    baskets[i] = -1;
                    --unplaced;
                    break;
                }
            }
        }
        return unplaced;
    }
};
